app/models/project.rb

```diff
@@ -34,15 +34,15 @@ def borough
   private
 
   def set_bbl_attributes
-    self.bbls_geom = CeqrData::Bbl.st_union_bbls(bbls)
-    self.bbls_version = CeqrData::Bbl.version
+    self.bbls_geom = Db::Bbl.st_union_bbls(bbls)
+    self.bbls_version = Db::Bbl.version
   end
 
   def create_analyses!
     # Create all analyses on creation of project, need better validations here.
```

Comment:
Do you have a preference on documentation style? At first I thought this comment was about the method call below, but it's actually about the `create_analyses!` method. I think method comments should go above the method their defining. Thoughts?

Reply:
Agree.